src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="forms-wrapper">
+      <form
+        action="/"
+        method="get"
+        data-qa="big"
+        class="form"
+      >
+        <label for="big-search" class="label-search-icon">
+          <img
+            class="icon"
+            src="images/Search.svg" alt="Search icon">
```

Comment:
Indent here should be similar to the one you have on the input below
```
<img
  class="icon"
  src="images/Search.svg" 
  alt="Search icon"
>
```

Reply:
I added correct indent.